steps/deploy-to-updraft/0.0.1/step.yml

```diff
@@ -0,0 +1,73 @@
+title: deploy-to-updraft
+summary: |
+  Distribute your app's builds to your team and testers
+description: |
+  This step will help you to upload your app's builds to Updraft. For more information please reach [Updraft](https://getupdraft.com) site.
+website: https://github.com/appswithlove/bitrise-step-deploy-to-updraft-android
+source_code_url: https://github.com/appswithlove/bitrise-step-deploy-to-updraft-android
+support_url: https://github.com/appswithlove/bitrise-step-deploy-to-updraft-android/issues
```

Comment:
I receive a 404 when opening these links. Could you check if these are correct?

Also in the project type tags you specify all platforms as supported but the link here points to the android step. Do you have dedicated steps per platform or a single one handling all of the supported platforms?

Reply:
Fixed